static/app/utils/profiling/hooks/useVirtualizedTree/useVirtualizedTreeReducer.tsx

```diff
@@ -48,6 +55,9 @@ export function VirtualizedTreeReducer<T>(
     case 'set scroll height': {
       return {...state, scrollHeight: action.payload};
     }
+    case 'set max scrollable height': {
+      return {...state, maxScrollableHeight: action.payload};
+    }
```

Comment:
I think I could have update the `set scroll height` to also handle for `maxScrollableHeight`. Not totally sure if this would be cleaner/better. Let me know if anyone prefers that instead.

Reply:
removed maxScrollableHeight from reducer. This can be calculate when needed by multiplying the flattened tree length by rowHeight